### PR TITLE
Adding Project.toml with compat bounds

### PR DESCRIPTION
**Pull request by theogf**

### CODE REVIEW

**Comment by theogf** [comment]
Should solve #4

**Comment by itsdfish** [comment]
Thank you for updating Project.toml. Would it be possible for you or someone else to release a new version with the changes? Unfortunately, this is preventing me from updating several packages. Thank you.

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name = "LNR"
+uuid = "7c4cb9fa-83a0-5386-9231-d6167c818060"
+
+[deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+Lazy = "50d2b5c4-7a5e-59d5-8109-a42b560f39c0"
+
+[compat]
+julia = "≥ 0.7.0"
```
Comment (devmotion) [inline Project.toml]: Compatibility bounds shouldn't be unbounded IMO, so
```suggestion
julia = "0.7, 1"
```
would be better.

**Comment by devmotion** [comment]
You could also remove the `REQUIRE` file.

**Comment by pfitzseb** [comment]
Thanks for the PR!

**Pull request merged**